frontend/www/js/omegaup/login/signin.ts

```diff
@@ -83,13 +95,26 @@ OmegaUp.on('ready', () => {
               ui.error(T.loginPasswordTooShort);
               return;
             }
-
-            api.User.create({
-              username: username,
-              email: email,
-              password: password,
+            const request: {
+              username: string;
+              email?: string;
+              dateOfBirth: Date;
+              parentEmail?: string;
```

Comment:
Ya encontré el motivo por el cual no te permite registrar usuarios menores a 13 años. Los parámetros que recibe la API son los  siguientes:
```suggestion
              birth_date: Date;
              parent_email?: string;
```